package.json

```diff
@@ -7,9 +7,10 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "@types/styled-components": "^5.1.26",
+    "dotenv": "^16.0.3",
```

Comment:
How can I use `env` vars in the component?

Reply:
I see the PR instructions
<img width="524" alt="Screenshot 2022-11-21 at 10 05 48" src="https://user-images.githubusercontent.com/14818162/202986273-a2300bd7-3066-4831-b299-10e494d27ccd.png">

But I don't see anything in the console
<img width="868" alt="Screenshot 2022-11-21 at 10 05 28" src="https://user-images.githubusercontent.com/14818162/202986397-21c7e67e-f371-4304-8863-c08af8f8976d.png">